Firestore/core/src/firebase/firestore/util/firebase_assert.h

```diff
@@ -35,14 +35,14 @@
 
 // Assert condition is true, if it's false log an assert with the specified
 // expression as a string.
-#define FIREBASE_ASSERT_WITH_EXPRESSION(condition, expression)               \
-  do {                                                                       \
-    if (!(condition)) {                                                      \
-      firebase::firestore::util::FailAssert(                                 \
-          __FILE__, __PRETTY_FUNCTION__, __LINE__,                           \
-          FIREBASE_EXPAND_STRINGIFY(expression));                            \
-    }                                                                        \
-  } while(0)
+#define FIREBASE_ASSERT_WITH_EXPRESSION(condition, expression) \
```

Comment:
This is a surprising whitespace change. Are you on clang-format 6?

Reply:
Yes, clang-format version 6.0.0 (tags/google/stable/2017-11-14).